call_apply_bind_exercise/callApplyBind.js

```diff
@@ -0,0 +1,93 @@
+///////////////////////////////////////////////////////////
+// 1: fix the following code
+///////////////////////////////////////////////////////////
+
+var obj = {
+    fullName: "Harry Potter",
+    person: {
+        sayHi: function(){
+            return "This person's name is " + obj.fullName;
+        }
+    }
+}
+
+// call the function: 
+
+obj.person.sayHi();
```

Comment:
This does not fix our issue :( - see what this returns and use `call` to fix it.